device-types/Zyxel/GS1920-48HPv2.yaml

```diff
@@ -0,0 +1,216 @@
+---
+manufacturer: Zyxel
+model: GS1920-48HPv2
+slug: zyxel-gs1920-48hpv2
+part_number: GS1920-48HPv2
+u_height: 1
+is_full_depth: false
+comments: '[Zyxel GS1920-48HPv2 48-port GbE Smart Managed Switch](https://www.zyxel.com/global/en/products/switch/8-24-48-port-gbe-smart-managed-switch-gs1920-series)'
+weight: 4.1
+weight_unit: kg
```

Comment:
Please add the `airflow` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not

Reply:
I currently have no way to verify the direction of airflow. It's probably side-to-side, but I don't know if it's right-to-left or left-to-right. Only info in datasheet is passive or active cooling.